### PR TITLE
WIP: tk-conform-education-00

### PR DESCRIPTION
**Pull request by kantrowitz**

### CODE REVIEW

File: _education/work-learn.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+  - url: /education/work-learn/workstudy/
+    title: Work Study
+    description: ""
+  - url: /education/work-learn/job-and-apprenticeship/
+    title: On-the-Job Training and Apprenticeships
+    description: ""
+  - url: /education/work-learn/co-op-training/
+    title: Co-op Training
+    description: ""
+ - url: /education/work-learn/non-college-degree-program/
```
Comment (emilyville) [inline _education/work-learn.md]: this needs one.more.space. :)

**Comment by kantrowitz** [comment]
@bethpotts Don't merge this yet. This will be the last one, once everything is okayed.

**Comment by kantrowitz** [comment]
@rcarthur

**Comment by emilyville** [comment]
Let's merge as we go? I prefer smaller changes going out more frequently when there is no "stakeholder review" component.

**Comment by emilyville** [comment]
@kantrowitz this looks good to me! I will go ahead and merge if we're not waiting on any further content review.

**Pull request merged**